extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -152,6 +152,12 @@ public static FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem
     public boolean validateAtStart;
 
+    /**
+     * true to automatically execute a Flyway clean command when a validation error occurs at start, false otherwise.
+     */
+    @ConfigItem(name = "validate-at-start.clean-on-validation-error")
```

Comment:
Is this config name correct?

Reply:
You're right, we shouldn't have a name there AFAICS.

@mschorsch could you create a follow-up PR to drop the name? Keep the `@ConfigItem` but remove the attributes.

Reply:
I haven't seen the discussion in https://github.com/quarkusio/quarkus/pull/46033#discussion_r1939820523 but I agree with @gsmet that the `ConfigItem` shouldn't have a name there

Reply:
@gsmet Does this mean that `clean-on-validation-error` should be executed independently of `validate-at-start`? If not, should it be renamed to `clean-on-validate-at-start-error` or similar?

Reply:
No need to. You can specify that behavior in the config javadoc.

Reply:
Well, I haven't followed this PR closely. My understanding is that we were restoring an existing behavior so I would keep the existing behavior.

Reply:
BTW, push back if you disagree as I really haven't digged into this myself.

/cc @yrodiere

Reply:
@gsmet the behavior we're introducing here is not exactly the same as the one we had in previous versions: it will only apply to validate-at-start, not to any validation.

That's why I suggested a different name.

See https://github.com/quarkusio/quarkus/pull/46033#discussion_r1939616584